src/dashboard/package.json

```diff
@@ -90,28 +90,28 @@
     "validator": "^13.7.0"
   },
   "devDependencies": {
+    "glob":"^9.0.0",
     "@types/history": "^4.7.2",
     "@types/react": "^16.8.1",
     "@types/react-dom": "^16.0.11",
-    "glob": "^9.0.0",
     "@umijs/preset-react": "^1.2.2",
     "antd-pro-merge-less": "^1.0.0",
     "antd-theme-webpack-plugin": "^1.2.0",
-    "babel-eslint": "^10.0.1",
+    "babel-eslint": "^10.1.0",
     "chalk": "^2.4.2",
     "check-prettier": "^1.0.1",
     "cross-env": "^5.2.0",
     "cross-port-killer": "^1.0.1",
     "enzyme": "^3.9.0",
-    "eslint": "^5.13.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-config-prettier": "^4.1.0",
-    "eslint-plugin-babel": "^5.3.0",
-    "eslint-plugin-compat": "^2.6.3",
-    "eslint-plugin-import": "2.20.1",
-    "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-markdown": "^2.2.0",
-    "eslint-plugin-react": "^7.12.4",
+    "eslint": "^8.57.1",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-babel": "^6.0.0",
```

Comment:
latest `eslint-plugin-babel` version is [5.3.1](https://www.npmjs.com/package/eslint-plugin-babel)